### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by hsoyier**
- removed vue-material

### CODE REVIEW

File: l [hunk 1]
```diff
@@ -0,0 +1,11 @@
+  animation[m
```
Comment (mbiemiller) [inline l]: is this supposed to be here?

File: src/components/TabDevelop.vue [hunk 2]
```diff
@@ -41,11 +41,11 @@
 
 <script lang="ts">
 import { Component, Prop, Vue, Watch } from "vue-property-decorator";
-// import { MdButton, MdContent, MdTabs } from "vue-material/dist/components";
+import { MdButton, MdContent, MdTabs } from "vue-material/dist/components";
```
Comment (mbiemiller) [inline src/components/TabDevelop.vue]: is this supposed to be here since we removed vue-material?

File: l (hunk 1 above)
Comment (hsoyier) [reply l]: sorry I didn't even make this file... weird lol just deleted.

File: src/components/TabDevelop.vue (hunk 2 above)
Comment (hsoyier) [reply src/components/TabDevelop.vue]: I also deleted TabDevelop since we never use this file.

**Pull request merged**